addon/templates/components/ember-popper.hbs

```diff
@@ -0,0 +1,9 @@
+{{#if renderInPlace}}
+  {{yield (hash popper=popper target=_popperTarget popperElement=_popperElement)}}
+{{else}}
+  {{#-in-element popperContainer}}
+    <div class="popper-{{elementId}} {{class}}">
+      {{yield (hash popper=popper target=_popperTarget popperElement=_popperElement)}}
+    </div>
+  {{/-in-element}}
```

Comment:
`{{#-in-element}}` puts the raw contents in its target, so we have to add a wrapper with a unique identifier so we can pass explicitly pass the element to the `Popper` for positioning